stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/view/activity/AttachmentDocumentActivity.java

```diff
@@ -65,17 +66,23 @@ private void configUIs() {
         webView.setWebViewClient(new AppWebViewClients());
     }
 
+    /**
+     * Encodes the give URL in a format that can be opened in the browser.
+     *
+     * @param url The URL of the file to load
+     * @return String representation of the encoded URL.
+     */
     private String encodeUrl(String url) {
         try {
-            return URLEncoder.encode(url, StandardCharsets.UTF_8.toString());
+            return URLEncoder.encode(url, StandardCharsets.UTF_8.name());
```

Comment:
Alternatively, we could use `Charset.defaultCharset().name()` here, but for Android, it's always `UTF-8` so this should be fine.